cdap-api/src/main/java/co/cask/cdap/api/plugin/PluginConfig.java

```diff
@@ -34,6 +35,10 @@
   private PluginProperties properties;
   private Set<String> macroFields;
 
+  public PluginConfig() {
+    this.macroFields = Collections.emptySet();
```

Comment:
I don't think that code is used to check that its called from unit test is it? It's just there as a workaround for unit tests?